docker-compose.yml

```diff
@@ -4,6 +4,8 @@ services:
       context: fastapi_backend
     environment:
       - OPENAPI_OUTPUT_FILE=./shared-data/openapi.json
+      - DATABASE_URL=postgresql+asyncpg://postgres:password@db:5432/mydatabase
```

Comment:
Having that here will prevent the user from needing to change that in the .env file, when running with docker.